src/System.Runtime.Serialization.Formatters/ref/project.json

```diff
@@ -1,5 +1,8 @@
 {
   "dependencies": {
+    "System.Collections.NonGeneric": "4.0.0",
```

Comment:
I believe @ericstj was telling me that this package is one of the few that we didn't actually ship and thus we don't need to rev the version number.  Eric, is that correct, or did I misunderstand?